packages/generator/templates/app/app/blitz-server.ts

```diff
@@ -0,0 +1,17 @@
+import {setupBlitz} from "@blitzjs/next"
+import {AuthServerPlugin, PrismaStorage} from "@blitzjs/auth"
+import {prisma as db} from "../db/index"
+import {simpleRolesIsAuthorized} from "@blitzjs/auth"
+
+const {gSSP, gSP, api} = setupBlitz({
+  plugins: [
+    AuthServerPlugin({
+      cookiePrefix: "webapp-cookie-prefix",
```

Comment:
We should have cookiePrefix: "__safeNameSlug__", which then will be replaced by the generator